eth/db/chain.py

```diff
@@ -184,32 +185,36 @@ def get_block_uncles(self, uncles_hash: Hash32) -> List[BlockHeader]:
         else:
             return rlp.decode(encoded_uncles, sedes=rlp.sedes.CountableList(BlockHeader))
 
-    # TODO: This method should take a chain of headers as that's the most common use case
-    # and it'd be much faster than inserting each header individually.
-    def persist_header(self, header: BlockHeader) -> Tuple[BlockHeader, ...]:
+    def persist_header(
```

Comment:
Should probably rename to `persist_chain()`, no? Also, why does it need to accept either a single header or an iterable?

Reply:
+1 to alternate name

- `persist_headers`
- `persist_header_chain` (my preference)